enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -105,9 +106,25 @@
    opts are passed to load-metabase"
   [path :- :string
    opts :- [:map
-            [:abort-on-error {:optional true} [:maybe :boolean]]
             [:backfill? {:optional true} [:maybe :boolean]]]]
-  (v2-load-internal! path opts :token-check? true))
+  (let [start    (System/currentTimeMillis)
+        err      (atom nil)
+        report   (try
+                   (v2-load-internal! path opts :token-check? true)
+                   (catch Exception e
+                     (reset! err e)))
+        imported (set (map (comp :model last) (:seen report)))]
+    (snowplow/track-event! ::snowplow/serialization-import nil
+                           {:source        "cli"
+                            :duration      (long (/ (- (System/currentTimeMillis) start) 1000))
+                            :models        (str/join "," imported)
+                            :count         (if (contains? imported "Setting")
+                                             (inc (count (remove #(= "Setting" (:model (first %))) (:seen report))))
+                                             (count (:seen report)))
```

Comment:
nitpick: This calculation is fairly non-trivial and repeated, I would pull out a helper method, but I know I'm more aggressive than others 😀

Reply:
😁 those two namespaces are not really connected and I did not want to press DRY too much here, honestly, analytics is maintained better when it's in-place in my opinion...

Reply:
Lack of a good place to put the helper is fair enough. It nice thing about the helper is that it would have forced a good name for the operation, which still slightly confuses me 😆